### PR TITLE
branch TCTC-01 code

### PR DESCRIPTION
**Pull request by Jasdeep-kaur8792**
TCTC-01 CODE 
LOGIN THE CLICK ON REPORT THEN LOGOUT

### CODE REVIEW

File: Config/testNg.xml [hunk 1]
```diff
@@ -7,11 +7,4 @@
             <class name="newTest.LogInTestNew"/>
         </classes>
     </test>
-
-<!--    <test name="Verify user clicked on leave page">-->
-<!--        <parameter name="browserName" value="chrome"/>-->
-<!--        <classes>-->
```
Comment (SachinKumar1606) [inline Config/testNg.xml:13]: Why commented code?

File: src/test/java/loginPageNEW/LogInPage.java [hunk 2]
```diff
@@ -15,4 +16,8 @@ public void newLogIn()  {
         driver.manage().window().maximize();
         driver.findElement(submit).click();
     }
+    public void newLogOut()  {
```
Comment (SachinKumar1606) [inline src/test/java/loginPageNEW/LogInPage.java:19]: Can we add space here

File: src/test/java/loginPageNEW/PageFactoryNew.java [hunk 3]
```diff
@@ -17,4 +18,11 @@ public LogInPage getLoginPage()
         }
         return loginPageNew;
     }
+    public Report getReport()
```
Comment (SachinKumar1606) [inline src/test/java/loginPageNEW/PageFactoryNew.java:21]: Space issue

File: src/test/java/loginPageNEW/PageFactoryNew.java [hunk 4]
```diff
@@ -5,6 +5,7 @@
 public class PageFactoryNew {
     WebDriver driver;
     private LogInPage loginPageNew;
+    private Report clickReport;
     public PageFactoryNew(WebDriver driver)
```
Comment (SachinKumar1606) [inline src/test/java/loginPageNEW/PageFactoryNew.java:9]: Space issue

File: src/test/java/loginPageNEW/Report.java [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+import org.openqa.selenium.WebDriver;
+
+public class Report {
+    WebDriver driver;
+    By clickReport = By.xpath("(//*[@class='nav-link active'])[1]");
+
+    public Report(WebDriver driver) {
+        this.driver = driver;
+    }
+    public void clickReport()  {
```
Comment (SachinKumar1606) [inline src/test/java/loginPageNEW/Report.java:13]: Space issue

File: src/test/java/loginPageNEW/Report.java [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+        driver.manage().window().maximize();
```
Comment (SachinKumar1606) [inline src/test/java/loginPageNEW/Report.java:14]: Why we writing it here can be write this in base class?

File: src/test/java/loginPageNEW/Report.java [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+        driver.findElement(clickReport).click();
+    }
+
```
Comment (SachinKumar1606) [inline src/test/java/loginPageNEW/Report.java:17]: Why extra line

File: src/test/java/newTest/LogInTestNew.java [hunk 8]
```diff
@@ -9,6 +9,8 @@ public class LogInTestNew extends Baseclass {
     public void start() throws InterruptedException
```
Comment (SachinKumar1606) [inline src/test/java/newTest/LogInTestNew.java:9]: What is Start can be rename the class to more descriptive so that we can identify the test

**Review by SachinKumar1606 — CHANGES_REQUESTED** [review]
No test found

**Comment by Jasdeep-kaur8792** [comment]
I will work on that points

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Pull request merged**